uat/custom-components/client-java-paho/src/main/java/com/aws/greengrass/testing/mqtt5/client/grpc/GRPCControlServer.java

```diff
@@ -168,8 +168,7 @@ public void createMqttConnection(MqttConnectRequest request,
             MqttLib.ConnectionParams.ConnectionParamsBuilder connectionParamsBuilder
                     = MqttLib.ConnectionParams.builder()
                     .clientId(clientId)
-                    .host("ssl://".concat(host))
-                    .port(port)
+                    .uri(createUri(host, port, request.hasTls()))
```

Comment:
createUri() is a paho-java specific, why it is located in grpc part of the code?

Reply:
fixed in cfc2de71bf01b7ff1f7ba96a1d547b69f5666bd1